### PR TITLE
Protecting against long-running queries

### LINKED ISSUE #787: How to protect against long-running queries
### What are the typical use cases?

Long running queries can hold locks, queue up WAL, or just consume a lot of system resources. Users would like an easy way to prevent queries from running on too long.

## Communication goals (e.g. detailed howto vs orientation)

Show people that the `statement_timeout` GUC automates aborting long queries.

### Corner cases, gotchas

Changing this value on the coordinator doesn't automatically propagate to Citus workers.
Note to self: test what happens if the value is set only on the coordinator.

### Are there relevant blog posts or outside documentation about the concept/feature?

* https://www.citusdata.com/blog/2017/04/28/postgres-tips-for-rails/
* https://www.postgresql.org/docs/10/static/runtime-config-client.html#GUC-STATEMENT-TIMEOUT

### PR DESCRIPTION
**Pull request by begriffs**
Fixes #787

### CODE REVIEW

File: admin_guide/cluster_management.rst [hunk 1]
```diff
@@ -406,6 +406,30 @@ Finally, by joining ``citus_stat_statements`` with ``pg_stat_statements`` we can
...
+
+Limiting Long-Running Queries
+-----------------------------
+
+Long running queries can hold locks, queue up WAL, or just consume a lot of system resources, so in a production environment it's good to prevent them from running too long. You can set the `statement_timeout <https://www.postgresql.org/docs/10/static/runtime-config-client.html#GUC-STATEMENT-TIMEOUT>`_ parameter on the coordinator and workers to cancel queries that run too long.
+
+.. code-block:: postgres
+
+   -- limit queries to five minutes
+   ALTER DATABASE citus
```
Comment (sumedhpathak) [inline admin_guide/cluster_management.rst]: Don't we cancel the worker queries if we set this on the coordinator? Did you observe any different behavior?

Comment (begriffs) [reply admin_guide/cluster_management.rst]: No, I observed the queries on the workers to keep running (with citus 7.5.0 on Cloud at least)

**Comment by sumedhpathak** [comment]
🚢 . Separately, @marcocitus Wouldn't statement_timeout on the coordinator cause cancellation of the worker queries?
